R/PNNL_DMS_utils.R

```diff
@@ -10,26 +10,38 @@
 #' * `get_dms_job_records()`: returns job records given variety keyword patterns
 #' * `get_tool_output_files_for_job_number()`: returns the output of the tool given tool name and file pattern
 #' * `get_output_folder_for_job_and_tool()`: returns the path given job id and tool name
-#'
-#' * `get_AScore_results()`: returns Ascore results given data packge number
-#' * `get_job_records_by_dataset_package()`: returns job records given data packge number
+#' * `get_AScore_results()`: returns Ascore results given data package number
+#' * `get_job_records_by_dataset_package()`: returns job records given data package number
 #' * `get_results_for_multiple_jobs()`: returns concatenated results given job numbers
 #' * `get_results_for_multiple_jobs.dt()`: returns results as concatenated data.table given job numbers
 #' * `get_results_for_single_job()`: returns results given job number
 #' * `get_results_for_single_job.dt()`: returns results as data.table given job number
 #' @md
 #'
 #'
-#' @param mtdbName (string) name of mass tag database
-#' @param data_package_num (interger) data package id
+#' @param data_package_num (integer) data package id
 #' @param jobs (integer) DMS job ID
-#' @param xPttrn (string) substrings for names of dataset, experiment, tool, parameter file, settings file, fasta file, protein options, instrument name
 #' @param mostRecent (logical) only most recent or all output files
+#' @param jobNumber placeholder
```

Comment:
do these placeholders need to be updated?

Reply:
Yes. I made sure that they are all called "placeholder" so they can be easily identified and changed.